README.md

```diff
@@ -1,19 +1,28 @@
-# Medical Text Classifier
+# medinify
 
-Text classification with a focus on medical text.
+Medical text extraction and classification 
 
 ## Requirements
 
 * Python 3.6
 * Cannot use Python 3.7 until TensorFlow supports it (12/2/18)
 
-## Installation
+## Getting Started
+
+To install medinify, run:
+
+```bash
+pip install git+https://github.com/NanoNLP/medinify.git
```

Comment:
Won't this line install medinify outside of the venv? I think `pip install -e .` is all we need if they're already cloning the repo.